Dockerfile

```
@@ -42,6 +42,17 @@ COPY --from=ui /out /app/ui/out
 COPY --from=provider-ui /out /app/provider-ui/out
 COPY --from=wrk2 /wrk2 /app/cmd/wrk2
 COPY --from=wrk2 /wrk2/wrk /usr/local/bin
+
+RUN curl -L -s `curl -s https://api.github.com/repos/DelusionalOptimist/wasm-filters/releases/latest | grep "browser_download_url" | cut -d : -f 2,3 | tr -d '"'` -o wasm-filters.tar.gz \

```

```suggestion
RUN curl -L -s `curl -s https://api.github.com/repos/layer5io/wasm-filters/releases/latest | grep "browser_download_url" | cut -d : -f 2,3 | tr -d '"'` -o wasm-filters.tar.gz \

```
Should this be layer5io ?

---

Thanks @sayantan1413.
Yes, you're right this should be. However we might need to think about some things before we publish these binaries at https://github.com/layer5io/wasm-filters.
So this is just a placeholder till then. For demo purposes.

---

Oh okay got it @DelusionalOptimist .✔️